src/Orleans/Serialization/RemoteNonDeserializableException.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Runtime.Serialization;
+using Orleans.Runtime;
+
+namespace Orleans.Serialization
+{
+    using System;
+    using System.Text;
+
+    /// <summary>
+    /// Represents an exception which cannot be fully deserialized.
+    /// </summary>
+    [Serializable]
+    public class RemoteNonDeserializableException : OrleansException
+    {
+        public RemoteNonDeserializableException() { }
```

Comment:
I'd think we'd at least need original exception's message, type, and call stack..?